src/styles/css/index.css

```
@@ -0,0 +1,265 @@
+@tailwind base;
```

Stylelint (17.3.0)

[error] Unexpected unknown at-rule "@tailwind" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)